pkg/apis/apps/v1beta2/conversion.go

```diff
@@ -118,6 +118,9 @@ func Convert_extensions_RollingUpdateDaemonSet_To_v1beta2_RollingUpdateDaemonSet
 }
 
 func Convert_v1beta2_RollingUpdateDaemonSet_To_extensions_RollingUpdateDaemonSet(in *appsv1beta2.RollingUpdateDaemonSet, out *extensions.RollingUpdateDaemonSet, s conversion.Scope) error {
+	if in.MaxUnavailable == nil {
```

Comment:
It can't be `nil` (and its value has to be >0)

Reply:
This should be removed, as `appsv1beta2.RollingUpdateDaemonSet` is never `nil` (based on default and validation).